docs/superpowers/plans/2026-05-26-httpclient-tracing.md

```
@@ -0,0 +1,1475 @@
+	})
+	require.NotNil(t, span)
+	assert.True(t, span.IsRecording(), "span should be recording with a real tracer provider")
+	span.End()
+}
+```
+
+- [ ] **Step 2: Run test — expect compile failure**
+
+```bash
+go test ./httpclient/internal/tracking/ -run TestStartHTTPClientSpanReturnsRecordingSpan -v
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)